src/sonic-yang-mgmt/yang-models/sonic-vlan.yang

```diff
@@ -75,8 +75,10 @@ module sonic-vlan {
 				}
 
 				leaf ip-prefix {
-					mandatory true;
```

Comment:
So this is not needed or we had too restricted before?

Reply:
Yang List Keys are mandotary by default. So this statement was redundant.